### DISCUSSION TITLE
can "exported junk data" be reused into BCU?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [Klocman/Bulk-Crap-Uninstaller#599] [Q&A]
hi!

i was uninstalling mysql etc, and on the junk cleaner (after finishing uninstalls) i successfully exported the data (using the export button) but thereafter accidently pressed escape.

so, now i have this `.txt` file containing that exported junk data. 
* can that be somehow fed into BCU? 
* or else, do u have any idea on how it can be used to cleanup?

sample of the exported data:

```
MySQL Workbench 8.0 CE | Oracle Corporation | 8.0.34 | MsiExec.exe /I{F7FFB7C8-1738-4FE0-A80B-61E5ED01B033} | MySQL Database Design + SQL Work Tool - Confidence: Good - HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\Folders => D:\ProgramFiles\MySQL\MySQL Workbench 8.0 CE\python\
MySQL Shell 8.1.1 | Oracle and/or its affiliates | 8.1.1 | MsiExec.exe /X{16CF8801-CD22-4820-B8A6-AEB69282D632} |  - Confidence: Questionable - C:\ProgramData\MySQL
MySQL Shell 8.1.1 | Oracle and/or its affiliates | 8.1.1 | MsiExec.exe /X{16CF8801-CD22-4820-B8A6-AEB69282D632} |  - Confidence: Questionable - D:\ProgramFiles\MySQL
MySQL Shell 8.1.1 | Oracle and/or its affiliates | 8.1.1 | MsiExec.exe /X{16CF8801-CD22-4820-B8A6-AEB69282D632} |  - Confidence: Questionable - D:\UserFiles\scoop\apps\mysql
```

putting in markdown headers from myself, this becomes:

name | corp | ver | uninstall cmd | junk
-- | -- | -- | -- | --
MySQL Workbench 8.0 CE | Oracle Corporation | 8.0.34 | MsiExec.exe /I{F7FFB7C8-1738-4FE0-A80B-61E5ED01B033} | MySQL Database Design + SQL Work Tool - Confidence: Good - HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\Folders => D:\ProgramFiles\MySQL\MySQL Workbench 8.0 CE\python\
MySQL Shell 8.1.1 | Oracle and/or its affiliates | 8.1.1 | MsiExec.exe /X{16CF8801-CD22-4820-B8A6-AEB69282D632} |  - Confidence: Questionable - C:\ProgramData\MySQL
MySQL Shell 8.1.1 | Oracle and/or its affiliates | 8.1.1 | MsiExec.exe /X{16CF8801-CD22-4820-B8A6-AEB69282D632} |  - Confidence: Questionable - D:\ProgramFiles\MySQL
MySQL Shell 8.1.1 | Oracle and/or its affiliates | 8.1.1 | MsiExec.exe /X{16CF8801-CD22-4820-B8A6-AEB69282D632} |  - Confidence: Questionable - D:\UserFiles\scoop\apps\mysql

**Comment by Klocman** [comment]
Hello!

It's not possible to re-import any exported data. You will have to manually remove the mentioned junk.
